GPU/GPUTracking/Base/GPUReconstructionConvert.cxx

```diff
@@ -273,25 +304,44 @@ void GPUReconstructionConvert::RunZSEncoder(const GPUTrackingInOutDigits* in, st
             tbHdr->rowAddr1()[l - 1] += 2 * nRowsInTB;
           }
         }
+        if (page && (k >= tmpBuffer.size() || endpoint != lastEndpoint)) {
+#ifdef GPUCA_O2_LIB
+          if (raw) {
+            raw->addData(rawfeeid, rawcru, rawlnk, rawendpoint, *ir + hbf * o2::constants::lhc::LHCMaxBunches, gsl::span<char>((char*)page + sizeof(o2::header::RAWDataHeader), (char*)page + TPCZSHDR::TPC_ZS_PAGE_SIZE), true);
```

Comment:
The passed size looks strange, should it be ``TPCZSHDR::TPC_ZS_PAGE_SIZE - sizeof(o2::header::RAWDataHeader)`` ?